doc/news/changes/minor/20180312MennoFraters

```diff
@@ -0,0 +1,4 @@
+New: JSON files which are written out by the
+parameter handler can be read in again.
```

Comment:
Can you reference the name of the function, in the form `ParameterHandler::parse_input_from_json()`? This way doxygen automatically links to the new function.